packages/paste-libraries/code-editor/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@twilio-paste/code-editor-library",
+  "version": "0.0.0",
+  "category": "library",
+  "status": "production",
+  "description": "A library to display and make changes to large blocks of code.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && NODE_ENV=production node build.js && tsc",
+    "build:js": "NODE_ENV=development node build.js",
+    "clean": "rm -rf ./dist",
+    "tsc": "tsc"
+  },
+  "dependencies": {
+    "@monaco-editor/react": "4.5.1",
```

Comment:
question: what tests does this package have? Are they worth copying? Reason I ask, is what tests do we need to know an upgrade of the package is "fine"? Do we need tests?

Reply:
it still renders and looks like an editor, which is covered by storybook vrt, then that integration test is sufficient